database/rrdhost.c

```diff
@@ -401,6 +401,33 @@ RRDHOST *rrdhost_find_or_create(
                 , system_info
                 , 0
         );
+
+        static int is_master_label_updated = 0;
+        if (!is_master_label_updated) {
+            struct label *prev_label = NULL;
+            struct label *label = localhost->labels;
+
+            uint32_t is_master_hash = simple_hash("_is_master");
+
+            netdata_rwlock_wrlock(&localhost->labels_rwlock);
+            while (label) {
+                if (label->key_hash == is_master_hash && !strcmp(label->key, "_is_master")) {
+                    struct label *new_label = create_label("_is_master", "true", LABEL_SOURCE_AUTO);
```

Comment:
There is no need to create a new node in the list - just overwrite the value.

Reply:
To overwrite the value I need to change your code to
```
struct label *create_label(char *key, char *value)
{
    struct label *label = callocz(1, sizeof(struct label));
    if (label != NULL) {
        label->key = strdupz(key);
        label->value = strdupz(value);
    }
    return label;
}
```

Reply:
No you do not - you just update the structure.

Reply:
Ok. "false" has more characters than "true". But it's a dirty hack to suppose that the code will never be changed. What if at some point somebody decides that words should be, for example, "yes" and "no"?

Reply:
That is a fair point - ignore my comment about updating the structure directly.